src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
There is no need to create a new `Lottery` object for each ball. Just create the object once and use it in a `for` loop with three iterations, so with the each iteration a new `Ball` would be created.